### PR TITLE
【Hackathon 6th No.10】Add isposinf / isneginf / isreal API to Paddle - part

### PR DESCRIPTION
**Pull request by NKNaN**
### PR Category

User Experience

### PR Types

New features

### Description

Add isposinf / isneginf / isreal
RFC：https://github.com/PaddlePaddle/community/pull/876

### CODE REVIEW

**Review by luotao1 — DISMISSED** [review]

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -7652,3 +7652,144 @@ def signbit(x, name=None):
...
+            [False, True, False, False, False])
+
+    """
+    if not isinstance(x, (paddle.Tensor, Variable, paddle.pir.Value)):
+        raise TypeError(f"x must be tensor type, but got {type(x)}")
+
+    check_variable_and_dtype(
+        x,
+        "x",
+        [
```
Comment (jeff41404) [inline python/paddle/tensor/math.py:7762]: shall we support data type of `bf16`?

File: python/paddle/tensor/math.py [hunk 2]
```diff
@@ -7652,3 +7652,144 @@ def signbit(x, name=None):
...
+            [False, False, False, True, False])
+
+    """
+    if not isinstance(x, (paddle.Tensor, Variable, paddle.pir.Value)):
+        raise TypeError(f"x must be tensor type, but got {type(x)}")
+
+    check_variable_and_dtype(
+        x,
+        "x",
+        [
```
Comment (jeff41404) [inline python/paddle/tensor/math.py:7809]: shall we support data type of `bf16`?

**Comment by jeff41404** [comment]
it seems that `isin` is not included in this PR, delete it from title or add this API?

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (NKNaN) [reply python/paddle/tensor/math.py:7762]: yes. 
bf16 type added.

File: python/paddle/tensor/math.py (hunk 2 above)
Comment (NKNaN) [reply python/paddle/tensor/math.py:7809]: yes. 
bf16 type added.

**Comment by NKNaN** [comment]
> it seems that `isin` is not included in this PR, delete it from title or add this API?

`isin` will be added in another PR.
Deleted.

**Comment by luotao1** [comment]
@NKNaN CI 再rerun下？

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**